packages/core/test/integration/support.ts

```diff
@@ -43,34 +43,33 @@ before(async () => {
   sequelize.hooks.addListener('afterQuery', (options, query) => {
     runningQueries.delete(query);
   });
+
+  // Reset the DB a single time for the whole suite
+  await clearDatabase();
```

Comment:
The problem this introduces is that before, it was guaranteed that the database was reset before each new suite:

- if setResetMode was not used, the default mode reset the database before each test
- if setResetMode was used, the database was reset before the tests in the suite were executed

With the current setup, the database is reset only once before any test starts, but there is no guarantee that the database will be clean when going from one suite to the next.

This is important: What `change-column.test.ts` puts in the database should not impact `create-table.test.ts`, but with the current approach, I don't know how you would do that, except having to use setResetMode at the beginning of every top-level describe block, and removing the default value